Dockerfile

```diff
@@ -1,19 +1,16 @@
-FROM ubuntu:bionic
+FROM ubuntu:bionic as oasis-chain-builder
 
 ENV DEBIAN_FRONTEND=noninteractive
 ENV RUSTUP_HOME=/usr/local/lib/rustup
 ENV CARGO_HOME=/usr/local/lib/cargo
 
-EXPOSE 8546/tcp
-
 RUN \
  apt-get update -q -q && \
  apt-get install --yes tzdata curl ca-certificates git build-essential && \
  echo 'UTC' > /etc/timezone && \
  rm /etc/localtime && \
  dpkg-reconfigure tzdata && \
  apt-get upgrade --yes && \
- adduser --system --group oasis --home /nonexistent && \
  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --no-modify-path --default-toolchain nightly && \
  RUSTFLAGS='-C target-feature=+aes,+ssse3' /usr/local/lib/cargo/bin/cargo install --locked --git https://github.com/oasislabs/oasis-chain.git oasis-chain && \
  cp /usr/local/lib/cargo/bin/oasis-chain /usr/local/bin/oasis-chain && \
```

Comment:
All lines from here to the end here you could just remove, no? To speedup building. We do not need cleanup anymore then now.

Reply:
ehh. it keeps the intermediate layers small 🤷‍♂️

Reply:
But one does not pull those layers when pulling the final image?